lib/tom_queue/delayed_job/worker.rb

```diff
@@ -0,0 +1,6 @@
+module TomQueue
+  module DelayedJob
```

Comment:
Should this be `TomQueue::Worker`?

I'm thinking the DelayedJob namespace could be used as the mock namespace that we map across to `Delayed` when the time comes?